common/src/main/resources/tachyon-default.properties

```diff
@@ -54,6 +54,7 @@ tachyon.integration.worker.resource.mem=1024MB
 
 # Security properties
 tachyon.security.authentication.type=NOSASL
+tachyon.security.authentication.socket.timeout_second=30
```

Comment:
how about rename this property to `tachyon.security.authentication.socket.timeout.second`?